website/common/locales/en/groups.json

```diff
@@ -446,6 +446,11 @@
   "worldBossBullet2": "The World Boss won’t damage you for missed tasks, but its Rage meter will go up. If the bar fills up, the Boss will attack one of Habitica’s shopkeepers!",
   "worldBossBullet3": "You can continue with normal Quest Bosses, damage will apply to both",
   "worldBossBullet4": "Check the Tavern regularly to see World Boss progress and Rage attacks",
+
+  "approvalNotRequired": "Approval not required",
+  "messagedFlaggedNotHidden": "Message flagged once, not hidden",
+  "messageHidden": "Message hidden",
+  "approvalNotRequired": "Approval not required",
```

Comment:
There's some duplication in these four strings and in the other three new ones at the bottom of this file.

Also all of these new strings won't ever need to be translated so they could be put into the file where non-translated strings are stored (I can't recall where that file is).